recipes/aneris/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.2.0" %}
+{% set sha256 = "624412e196fcefc2708d472c6840509492f91327e1fabcaf46fe61ab9a0705ff" %}
+
+package:
+  name: aneris
+  version: {{ version }}
+
+source:
+  url: https://github.com/iiasa/aneris/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - aneris = aneris.cli:main
+
+requirements:
+  build:
+    - python
```

Comment:
I think you need to pin the Python version if you're going to use noarch, see https://github.com/conda-forge/staged-recipes/pull/6830/files#r225533181